fcrepo-http-api/src/main/java/org/fcrepo/http/api/ContentExposingResource.java

```diff
@@ -614,4 +617,37 @@ private static URI checksumURI( final String checksum ) {
         }
         return null;
     }
+
+    protected void createFixityEvent(final EventBus eventBus, final RdfStream rs,
+                                     final String baseURL, final String agent,
+                                     final String userID) {
+        final FixityEvent fixityEvent = new FixityEvent();
+        fixityEvent.setDate(System.currentTimeMillis());
+        fixityEvent.setType(128);
```

Comment:
See http://www.day.com/maven/jsr170/javadocs/jcr-2.0/constant-values.html#javax.jcr.observation.Event.NODE_ADDED -- the highest builtin type value is 64, so 128 would be the next value.